aten/src/ATen/native/BlasKernel.cpp

```diff
@@ -356,4 +358,84 @@ INSTANTIATE_VDOT_IMPL(c10::complex<double>);
 
 #undef INSTANTIATE_DOT_IMPL
 
+namespace blas_impl {
+
+template <typename scalar_t>
+void addr_naive(int64_t m, int64_t n, scalar_t alpha, scalar_t *x, int64_t incx, scalar_t *y, int64_t incy, scalar_t *a, int64_t lda) {
```

Comment:
Add a comment here with two parts:

- NOTE of when this is triggered
- brief description of the function

Reply:
Sure, I will address this in the later commit of this PR.